apps/portal/redirects.mjs

```diff
@@ -1,1098 +1,1113 @@
-  "/pre-built-contracts/airdrop-erc721":
-    "/contracts/explore/pre-built-contracts/airdrop-erc721",
-  "/pre-built-contracts/airdrop-erc721-claimable":
-    "/contracts/explore/pre-built-contracts/airdrop-erc721-claimable",
-  "/pre-built-contracts/airdrop-erc1155":
-    "/contracts/explore/pre-built-contracts/airdrop-erc1155",
-  "/pre-built-contracts/airdrop-erc1155-claimable":
-    "/explore/pre-built-contracts/airdrop-erc1155-claimable",
-  "/pre-built-contracts/edition":
-    "/contracts/explore/pre-built-contracts/edition",
-  "/pre-built-contracts/edition-drop":
-    "/contracts/explore/pre-built-contracts/edition-drop",
-  "/pre-built-contracts/how-it-works": "/contracts",
-  "/pre-built-contracts/loyalty-card":
-    "/contracts/explore/pre-built-contracts/loyalty-card",
-  "/pre-built-contracts/managed-account-factory":
-    "/contracts/explore/pre-built-contracts/managed-account-factory",
-  "/pre-built-contracts/marketplace":
-    "/contracts/explore/pre-built-contracts/marketplace",
-  "/pre-built-contracts/multiwrap":
-    "/contracts/explore/pre-built-contracts/multiwrap",
-  "/pre-built-contracts/nft-collection":
-    "/contracts/explore/pre-built-contracts/nft-collection",
-  "/pre-built-contracts/nft-drop":
-    "/contracts/explore/pre-built-contracts/nft-drop",
-  "/pre-built-contracts/open-edition-erc721":
-    "/contracts/explore/pre-built-contracts/open-edition",
-  "/pre-built-contracts/pack": "/contracts",
-  "/pre-built-contracts/signature-drop": "/contracts",
-  "/pre-built-contracts/split": "/contracts/explore/pre-built-contracts/split",
-  "/pre-built-contracts/stake-erc20":
-    "/contracts/explore/pre-built-contracts/stake-erc20",
-  "/pre-built-contracts/stake-erc721":
-    "/contracts/explore/pre-built-contracts/stake-erc721",
-  "/pre-built-contracts/stake-erc1155":
-    "/contracts/explore/pre-built-contracts/stake-erc1155",
-  "/pre-built-contracts/token": "/contracts/explore/pre-built-contracts/token",
-  "/pre-built-contracts/token-drop":
-    "/contracts/explore/pre-built-contracts/token-drop",
-  "/pre-built-contracts/vote": "/contracts/explore/pre-built-contracts/vote",
-  // pack contract deprecation redirect
-  "/contracts/explore/pre-built-contracts/pack": "/contracts",
-  //publish
-  "/publish": "/contracts/publish/overview",
-  "/publish/deployment-options": "/contracts/publish/publish-options",
-  "/publish/get-featured-on-explore": "/contracts/publish/overview",
-  "/publish/getting-started": "/contracts/publish/publish-contract",
-  "/sdk": "/contracts",
-  "/sdk/getting-started": "/contracts",
-  "/sdk/how-it-works": "/contracts",
+	"/contracts/DropERC721": "/contracts/explore/pre-built-contracts/nft-drop",
+	"/contracts/DropERC1155":
+		"/contracts/explore/pre-built-contracts/edition-drop",
+	// DEPRECATED: Pre-built modular contracts redirects
+	"/contracts/explore/pre-built-modular/token":
+		"/contracts/explore/pre-built-contracts/token",
```

Comment:
@saminacodes these links are not right? gives me a 404

Reply:
https://portal.thirdweb.com/contracts/explore/pre-built-contracts/token try this?